README.md

```diff
@@ -1,24 +1,28 @@
-LLVM "Nightly Test" Infrastructure
-==================================
+LNT: LLVM "Nightly Test" Infrastructure
+=======================================
 
-This directory and its subdirectories contain the LLVM nightly test
-infrastructure. This is technically version "4.0" of the LLVM nightly test
-architecture.
+About
+=====
 
-The infrastructure has the following layout:
+*LNT* is an infrastructure for performance testing. The software itself
+consists of two main parts, a web application for accessing and visualizing
+performance data, and command line utilities to allow users to generate and
+submit test results to the server.
 
- $ROOT/lnt - Top-level Python 'lnt' module
+The package was originally written for use in testing LLVM compiler
+technologies, but is designed to be usable for the performance testing of any
+software.
 
- $ROOT/lnt/server/db - Database schema, utilities, and examples of the LNT plist format.
 
- $ROOT/docs - Sphinx documentation for LNT.
+Documentation
+=============
 
- $ROOT/tests - Tests for the infrastructure.
+The official *LNT* documentation is available online at:
+  https://llvm.org/docs/lnt
 
-For more information, see the web documentation, or docs/.
 
-Testing
-=======
+Source
+======
 
-Testing is done by running tox from the top-level directory. It runs the tests
-for Python 3 and checks code style.
+The *LNT* source is available in the llvm-lnt repository:
+  https://github.com/llvm/llvm-lnt
```

Comment:
I assume this was inspired by the other changes, but should be its own PR.

Reply:
Which is in the developer guide. Add a link from the readme to the developer guide.

Reply:
But a link to the developer guide would be nice, assuming it doesn't confuse the build somehow. Do that whenever though.